tests/requirements.txt

```diff
@@ -7,7 +7,7 @@
 astroid==2.1.0            # via pylint
 atomicwrites==1.2.1       # via pytest
 attrs==18.2.0             # via pytest
-certifi==2018.11.29       # via requests
```

Comment:
are you degrading `certifi` from `2018.11.29 to 2018.10.15`

Reply:
yes.CI/CD is failing because of that. ref: https://ci.centos.org/job/devtools-fabric8-analytics-worker-f8a-build-master/lastFailedBuild/console

Reply:
+1